array.js

```diff
@@ -0,0 +1,26 @@
+function selectionSort(arr) {
```

Comment:
Can u Please implement Quick Sort instead of Selection sort

Reply:
done

Reply:
Looks good, please add some documentation at the top(js docs)